pkg/api/controller.go

```diff
@@ -1946,6 +1947,14 @@ func (c *Controller) CreateRepository(w http.ResponseWriter, r *http.Request, bo
 		return
 	}
 
+	allowInterRegionStorage := c.Config.Installation.AllowInterRegionStorage
+	if !allowInterRegionStorage {
```

Comment:
I feel like this check should be inside validateInterRegionStorage

Reply:
Interesting.
I don't really mind actually, both seem ok to me 😄